src/reducers/entryDraft.js

```diff
@@ -73,6 +74,10 @@ const entryDraftReducer = (state = Map(), action) => {
     case REMOVE_ASSET:
       return state.update('mediaFiles', list => list.filterNot(path => path === action.payload));
 
+    case UNPUBLISHED_ENTRY_REGISTER_DEPENDENCY:
+      return state.setIn(['entry', 'data', 'dependencies', action.payload.field],
```

Comment:
Ah, thanks for clarifying that. We definitely don't want this clashing with frontmatter.